homeassistant/components/amcrest/camera.py

```diff
@@ -4,6 +4,7 @@
 from functools import partial
 import logging
 import time
+import homeassistant.helpers.config_validation as cv
```

Comment:
This line should go before:
```python
from homeassistant.helpers.dispatcher import async_dispatcher_connect
```

Reply:
Yes, thanks - I had just figured that out from the raw log and had already moved the statement.

Yes, I am doing commits directly from github.   And as you can surmise from all the previous failed commits, my journey with trial and error fixing lint, flake, black has been an interesting one.   

I don't have a typical dev environment, but it sure looks like I am needing to go that way if I am ever to improve my progress with component development.    This was just me working from home playing around to see if I could accomplish some useful new feature  that I might interest you in incorporating into a later PR.   I really wasn't prepared to take on a new PR with all the development requirements that entails, but am starting to pick up this stuff as I go, bit by bit.

Reply:
Absolutely no problem. I started kind of the same way, but you seem to be progressing way faster than I did. lol! BTW, when I said it was best for you to check the changes into a branch on your own fork, you could have stopped there and just sent me a link to it. But it's good that you did submit the PR. You might as well get credit.